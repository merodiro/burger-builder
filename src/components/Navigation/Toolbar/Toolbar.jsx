import React from 'react'

import classes from './Toolbar.module.css'
import Logo from '../../Logo/Logo'

const Toolbar = ({ children }) => (
  <header className={classes.Toolbar}>
    <div>MENU</div>
    <Logo />
    <nav>...</nav>
  </header>
)

export default Toolbar
