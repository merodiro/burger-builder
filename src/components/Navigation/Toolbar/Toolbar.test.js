import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import Toolbar from './Toolbar'

describe('<Toolbar>', () => {
  it('renders the same output', () => {
    const component = shallow(<Toolbar />)
    expect(component).toMatchSnapshot()
  })
})
