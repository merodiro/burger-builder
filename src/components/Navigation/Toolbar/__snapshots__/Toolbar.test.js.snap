// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar> renders the same output 1`] = `
<header
  className="Toolbar"
>
  <div>
    MENU
  </div>
  <Logo />
  <nav>
    ...
  </nav>
</header>
`;
